src/main/resources/META-INF/primefaces-p.taglib.xml

```diff
@@ -26310,15 +26310,6 @@
             <required>false</required>
             <type>java.lang.Object</type>
         </attribute>
-        <attribute>
-            <description>
-                <![CDATA[Time zone the user's browser / PC is running in. This time zone allows to correct the conversion of start / end dates to the target timeZone for displaying.
-                The attribute can be either a String or TimeZone object or null. Note: browserTimeZone should be provided if the target timeZone is provided. If null, browserTimeZone defaults to the server's timeZone.]]>
-            </description>
-            <name>browserTimeZone</name>
-            <required>false</required>
-            <type>java.lang.Object</type>
-        </attribute>
```

Comment:
I removed this, because it´s not necessary anymore.
LocalDateTime-values are sent from the server to the client as `ISO_OFFSET_DATE_TIME`.
eg `new Date('2019-10-29T12:00:00+01:00'), end: new Date('2019-10-29T17:00:00+01:00')`
They are sent from the client to the server as isoString (`item.start.toISOString()`)
eg `2019-10-29T20:30:00.000Z`
IMO browserTimeZone was some kind of work-around because until now java.util.Date-values were transfered without any timezone-info.
eg "start":new Date(1572387021292),"end":new Date(1572415821292)
IMO the new solution is better than the old one. The browser know´s his timezone. He does not need some kind of awkward help from the server-side.

Reply:
I understand your solution, is seems a good solution for me and I'm OK with that. My recommendation was only to don't remove the property `browserTimeZone` from taglib file in order to avoid breaking existing xhtml that already uses that property and document the deprecation in the taglib instead of removing the property. What do you think @melloware @tandraschko ?

Reply:
I think its OK to mark it in the documentation as deprecated but can probably be removed from the taglib if it truly is not doing any good.  Anybody that breaks will be forced to lookup the documentation.

Reply:
Sorry I meant Migration Guide!